apps/desktop/src/app/sync/sources/folder.ts

```diff
@@ -12,15 +12,15 @@ import { Observable, BehaviorSubject, of, throwError } from 'rxjs';
 type ElectronFile = File & { relativePath: string };
 const FILES_TO_IGNORE = ['.DS_Store', 'Thumbs.db'];
 
-export const FolderConnector: SourceConnectorDefinition = {
-  id: 'folder',
-  title: 'Folder',
-  logo: 'assets/logos/folder.svg',
-  description: 'Upload a folder from your device',
-  factory: () => of(new FolderImpl()),
-};
+// export const FolderConnector: SourceConnectorDefinition = {
+//   id: 'folder',
+//   title: 'Folder',
+//   logo: 'assets/logos/folder.svg',
+//   description: 'Upload a folder from your device',
+//   factory: () => of(new FolderImpl()),
+// };
```

Comment:
dead code

Reply:
Only Dropbox and OneDrive are relevant for now, I just want to keep the old code